pyramid-app/{{ cookiecutter.slug }}/{{ cookiecutter.package_name }}/app.py

```diff
@@ -1,9 +1,20 @@
+from os import environ
+
 from pyramid.config import Configurator
 from pyramid.view import view_config
 
 
 def create_app(_=None, **settings):
     with Configurator(settings=settings) as config:
+        config.registry.settings["database_url"] = environ["DATABASE_URL"]
+
+        config.registry.settings["tm.annotate_user"] = False
+        config.registry.settings["tm.manager_hook"] = "pyramid_tm.explicit_manager"
+        config.include("pyramid_tm")
+
+        config.include("{{ cookiecutter.package_name }}.models")
+        config.include("{{ cookiecutter.package_name }}.db")
```

Comment:
Some of this is necessary for `make_db` to work, for example the `config.include("{{ cookiecutter.package_name }}.models")`. The rest of this isn't necessary for `make_db` but is necessary to integrate the SQLAlchemy session/transaction from `db.py` into Pyramid.

It would be perverse for the cookiecutter to generate `db.py` and `app.py` files but omit the few `app.py` lines that're necessary to integrate `db.py` into `app.py`.

This `app.py` template is only used when generating new Pyramid apps, `make template` doesn't update `app.py` in existing apps.